test/library_test.go

```diff
@@ -0,0 +1,33 @@
+package test
+
+import (
+	. "github.com/onsi/ginkgo/extensions/table"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestAPIUsageAsLibrary(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Library Suite")
+}
+
+var _ = DescribeTable("Using service binding API as library", func(projectPath string) {
+
+	cmd := exec.Command("sh", "-c", "go mod download; go mod tidy; go build -o main")
```

Comment:
If one of these commands returns an error, do we want to run successive commands?  For example, if `go mod download` or `go mod tidy` fails, building doesn't make sense, and I think we should short circuit.
```suggestion
	cmd := exec.Command("sh", "-c", "go mod download && go mod tidy && go build -o main")
```